HACKING.md

```diff
@@ -41,50 +43,47 @@ To check if the watcher daemon is running, use `docker ps`. Then you can use `do
 
 ### Running the site with native python
 
-Since the site is basically a Django app, you can also run the site in the traditional way using [python 2.7](https://www.python.org/download/releases/2.7/) and [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/):
+Since the site is basically a Flask app, you can also run the site in the traditional way using [python 3](https://docs.python.org/3/) and [venv](https://docs.python.org/3/library/venv.html?highlight=venv#module-venv):
 
 ``` bash
-virtualenv env
+python3 -m venv env
 source env/bin/activate
 pip install -r requirements.txt
 ./entrypoint 0.0.0.0:8001
 ```
 
 Now browse to the site at <http://127.0.0.1:8001>. If it looks a bit odd, it's probably because you haven't built sass - see below.
 
+You can run `deactivate` to shutdown the virtual environment when you are done.
+
 ### Building Sass
 
-The CSS needs to be built from the `static/css/styles.scss` file. This in turn requires [ubuntu-vanilla-theme](https://github.com/ubuntudesign/ubuntu-vanilla-theme) and its dependency, [vanilla-framework](https://github.com/vanilla-framework/vanilla-framework).
+The CSS needs to be built from the `static/css/styles.scss` file. This in turn requires [vanilla-framework](https://github.com/canonical-web-and-design/vanilla-framework).
 
-If you can't build using the `./run build` command, you can pull down dependencies this using `npm` or `yarn`:
+If you can't build using the `./run build` command, you can pull down dependencies this using `yarn`:
 
 ``` bash
-npm install
-# or
 yarn install
 ```
 
 Then you can use the built in scripts to build or watch the Sass:
 
 ``` bash
-npm run build  # Build the Sass to CSS then exit
-npm run watch  # Dynamically watch for Sass changes and build CSS
-# or
 yarn run build  # Build the Sass to CSS then exit
 yarn run watch  # Dynamically watch for Sass changes and build CSS
 ```
 
-### Overriding NPM modules
+### Overriding Yarn modules
 
-You can use the `./run` script to use NPM modules from a local folder on a one-time basis, instead of the modules declared in `package.json`, as follows:
+You can use the `./run` script to use Yarn NPM modules from a local folder on a one-time basis, instead of the modules declared in `package.json`, as follows:
```

Comment:
```suggestion
You can use the `./run` script to use Yarn modules from a local folder on a one-time basis, instead of the modules declared in `package.json`, as follows:
```